doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -37,6 +37,111 @@ It is recommended to follow the “url” link values,
 to keep your client decoupled from implementation details.
 
+## JSON entity representation
+
+Many operations use the same JSON entity representation format which is described in the present section. An
+entity is represented by a a JSON object which contain the following properties:
+
+* `id` for the entity ID, represented by a JSON string. If the entity has an `id` attribute itself it will be ignored.
+* `type` for the entity type, represented by a JSON string. If the entity has a `type` attribute itself it will be ignored.
+  If the entity doesn't have a type, then this property is not included.
+* A collection of properties, one per entity attribute. Property values can be single values or they can include,
+  `type` and properties representing metadata as well. Property values which correspond to the primitive JSON datatypes
+  "string", "number" and "boolean" and don't include metadadta are rendered as regular JSON values. Property values
+  which specify a custom type or include metadata are rendered as JSON objects with the following properties:
+  * `value`: for the attribute value, rendered as a regular JSON value depending on the type. Attributes with an
+    user-defined informative types are rendered as JSON strings.
+  * `type`: a JSON string which represents the attribute type provided at attribute creation time.
+  * A collection of properties one per metadata field associated to the attribute value. Metadata properties follows the
+    same representation rules that applies to attributes properties (except that a metadata
+    cannot have metadata properties). 
+
+Canonical format description (TBD).
+
+### Predefined types
+
+All attribute types not included in the list below are considered *user-defined informative* types. The values of
+user-defined informative types are represented by JSON string.
+
+* `number` (or `Number`):  identifies attribute meaning numbers, either integer or float (see https://developer.mozilla.org/en-US/docs/Glossary/Number).
+They are associated to JSON number type. They can be used in greater than, lesser than, greater or equal,
+lesser or equal and range filters.
+
+```
+"temperature": 32.5
+```
+
+* `string` (or `String`), which represents text string. They are represented by JSON strings.
+
+```
+"msg": "Hello world"
+```
+
+* `boolean` (or `Boolean`), which represent a logical value (either true or false). They are represented by JSON booleans.
+
+```
+"active": true
+```
+
+* `date` (or `Date`):  identifies attribute meaning a date, in ISO8601 format. They are represented by JSON strings. They
+can be used in greater than, lesser than, greater or equal, lesser or equal and range filters.
+
+```
+"timestamp": "2017-06-17T07:21:24.238Z"
+```
+
+
+* `geo:point`: identifies an attribute meaning a geolocation. They are represented by JSON strings. An
+entity can only have one `geo:point` attribute (TBD: this is yet under discussion, one possibliity is
```

Comment:
nit: possibility 

i'm wondering if we can mark issues with an special style in the document